### PR TITLE
Updated CORS headers

### PR DESCRIPTION
**Pull request by Pchelolo**
Bug: https://phabricator.wikimedia.org/T148368

Also updated one test page location, it's been deleted on wikipedia, moved it to labs.

cc @wikimedia/services

### CODE REVIEW

**Comment by berndsi** [comment]
This change should also be made to the node service template, so it can be picked up by other services, like MCS.

**Comment by Pchelolo** [comment]
@berndsi Yep, that's the plan after we discuss this PR

File: lib/security_response_header_filter.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ module.exports = function addCSPHeaders(hyper, req, next, options) {
         // Set up basic CORS headers
         rh['access-control-allow-origin'] = '*';
         rh['access-control-allow-methods'] = 'GET';
-        rh['access-control-allow-headers'] = 'accept, content-type';
+        rh['access-control-allow-headers'] =
+            'accept, accept-encoding, origin, accept-language, content-type';
```
Comment (gwicke) [inline lib/security_response_header_filter.js]: It's actually impossible to override `origin` in clients, as it is part of verifying the domain comes from as part of [the CORS system](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS). Adding it here would just be confusing.

Comment (Pchelolo) [inline lib/security_response_header_filter.js]: @gwicke Safari 9 sends it with an OPTIONS request. As I understand, normally you need to send back all the headers from OPTIONS preflight request to allow it and I couldn't find anything about `Origin` header being somehow specific

**Pull request merged**